0_RootFS/gcc_common.jl

```diff
@@ -352,6 +357,12 @@ function gcc_script(compiler_target::Platform)
             GLIBC_CONFIGURE_OVERRIDES+=( libc_cv_ssp=no libc_cv_ssp_strong=no )
         fi
 
+        if [[ ${COMPILER_TARGET} == riscv64-* ]]; then
+            # Explicitly disable C++
+            # (Disable for all architectures?)
+            GLIBC_CONFIGURE_OVERRIDES+=( CXX=false )
+        fi
```

Comment:
Why this?  What are the consequences?

Reply:
Without this setting, glibc builds the file `support/links-dso-program.cc` with a C++ compiler. There is no C++ compiler for the host architecture available, so glibc uses `/usr/bin/g++` instead. This fails. If we disable C++ (`/bin/false` always reports an error in the configure stage), then glibc builds a different version of that file (`support/links-dso-program-c.c`) with a C compiler, and all is fine.

Reply:
This file doesn't exist in the earlier glibc versions that we build for other architectures.

Reply:
> There is no C++ compiler for the host architecture available

All this stuff is built with system compilers, we don't use BinayBuilder toolchains, that's the point of these (obscure) lines https://github.com/JuliaPackaging/Yggdrasil/blob/b196211d789606f65ea85a0624c7fd4422c9a51b/0_RootFS/gcc_common.jl#L31-L32  The C file is probably being built with `/usr/bin/gcc`.